Source/UI/NewUI/PaymentForm/Factory/Theme/Theme.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  Theme.swift
+//  
+//
+//  Created by Alex Ioja-Yang on 08/08/2022.
+//
+
+import UIKit
+
+/// Template theme generating UI components for your layout
+public struct Theme {
+
+    // MARK: Font colors
+    /// Font color associated with text displaying primary font colour, like inputs, titles and other important text
+    public var primaryFontColor: UIColor
+    /// Font color associated with text displaying secondary information, like subtitles and other non core text
+    public var secondaryFontColor: UIColor
+    /// Font color associated with text displayed inside actionable buttons
+    public var buttonFontColor: UIColor
+    /// Font color associated with text
+    public var errorFontColor: UIColor
+
+    // MARK: Font sizes
+    /// Font size used for page headers
+    public var headerFontSize = UIFont.preferredFont(forTextStyle: .title1).pointSize
+    /// Font size used for input sections titles
+    public var titleFontSize = UIFont.preferredFont(forTextStyle: .subheadline).pointSize
+    /// Font size used for text displaying secondary information, like subtitles and other non core text
+    public var subtitleFontSize = UIFont.preferredFont(forTextStyle: .footnote).pointSize
+    /// Font size used for input fields where user is inputting the values
+    public var inputFontSize = UIFont.preferredFont(forTextStyle: .subheadline).pointSize
+    /// Font size used for buttons that the user will interact with
+    public var buttonsFontSize = UIFont.preferredFont(forTextStyle: .body).pointSize
+
+    // MARK: UI Formatting colors
+    /// Color used as background on screens
+    public var backgroundColor: UIColor
```

Comment:
```suggestion
    public var backgroundColor: UIColor = .clear
```

Reply:
I do not believe this is a reasonable default, please see property description. The background of the screens should be provided as part of the minimum, whether its white, black or pink.